diff --git a/native/src/CMakeLists.txt b/native/src/CMakeLists.txt
index b5628f9..f8283ac 100644
--- a/native/src/CMakeLists.txt
+++ b/native/src/CMakeLists.txt
@@ -31,7 +31,9 @@ else()
 endif()

 # Required files and directories
-include(GNUInstallDirs)
+if (NOT CMAKE_INSTALL_LIBDIR OR NOT CMAKE_INSTALL_INCLUDEDIR)
+    include(GNUInstallDirs)
+endif()
 set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${SEAL_SOURCE_DIR}/../lib)
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${SEAL_SOURCE_DIR}/../lib)
 set(CMAKE_LIBRARY_RUNTIME_DIRECTORY ${SEAL_SOURCE_DIR}/../bin)
